dynamic-simulation/dynamic-simulation-api/src/main/java/com/powsybl/dynamicsimulation/DynamicSimulation.java

```diff
@@ -71,6 +71,11 @@ public DynamicSimulationResult run(Network network, CurvesProvider curvesProvide
             return runAsync(network, curvesProvider, workingVariantId, computationManager, parameters).join();
         }
 
+        public DynamicSimulationResult run(Network network, CurvesProvider curvesProvider, ComputationManager computationManager,
+            DynamicSimulationParameters parameters) {
+            return run(network, curvesProvider, network.getVariantManager().getWorkingVariantId(), computationManager, parameters);
+        }
+
```

Comment:
@marcosmc In this class we would like to have all the combinations of possible calls of run and runAsync. Could you add them? There are 11 possibilities aside from the one you've done here to provide. Let me know if it's unclear to you.

Reply:
Done